tasks/rss-mentor-dashboard.md

```diff
@@ -1,77 +1,83 @@
 
 
-### Ограничения
-  - Должно работать в Chrome
-  - Можно гуглить, общаться между собой и пользоваться stackoverflow
-  - можно использовать lodash
-  - задание должно использовать React
-    
-### Как делать / сдавать
-  - написать nodejs скрипт который
-    - обработает данные с помощью библиотеки [node-xlsx](https://github.com/mgcrea/node-xlsx) из [папки с данными](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW) . Можно использовать любую другую библиотеку для чтения excel файлов.
-    - будет использовать файлы для обработки библиотекой - Tasks, Mentor score, Mentor-students pairs (информация по парам ментор - студент + информация по соотвествию имени / фамилии ментора его гитхаб аккаунту ) 
-    - исправит возникающие ошибки в данных (неправильное написание гитхаб аккаунтов, неправильные пары ментор-студент). Надо самому решать что исправлять скриптом, а что руками в исходных файлах. Если правятся исходные файлы - надо указать в readme при сдаче таска
-    - соединит (смержит, заджоинит (join)) данные из [google drive папки](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW)
-      - по студентам ментора
-      - по выставленным оценкам
-    - получит json файл с нужными данными
-    - загрузит данные на html страницу
-  - с помощью react и react component нарисовать дашборд, используя получившийся json
-  - сбилдить веб паком и выложить сбилженные файлы на github pages 
-  - pull request должен содержать 
-    - nodejs script, инструкцию как им пользоваться, 
-    - получившийся json
-    - js файл с реакт компонентой
-    - html страницу, которая подключает реакт, js файл с реактом, загружает json файл с данными, которые отображает react компонента 
-  - запостить ссылку на гитхаб страницу + PR [в форму](https://docs.google.com/forms/d/e/1FAIpQLSctLqyRXrpdgC1vybGjGZoiYwZUUAf6_hSD-gPTZMORS_E1kQ/viewform)
-    
+### Requirements
+  - Should work in Chrome
+  - You can Google, communicate, chat and use Stackoverflow
+  - You may use Lodash
+  - Task should use React
     
-### Требования к коммитам, PR, репозиторию
-[Общие требования из этапа 2](https://github.com/rolling-scopes-school/docs/blob/master/stage2-tasks-requirements.md)
+### How to do / send
+  - make nodejs script, which
+    - process data with help of library [node-xlsx](https://github.com/mgcrea/node-xlsx) from [data folder](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW).
+    You may use any other library for reading excel files.
+    - uses files for processing - Tasks, Mentor score, Mentor-students pairs
+    (information about pairs mentor - student + mapping name / surname of mentor to his github account)
+    - fixes errors in data (wrong github account, incorrect pairs mentor-student).
+    You should decide on your own what should be fixed by script and what manually in source files. If you change source files - add note about it in readme when finish task.
```

Comment:
"It is up to you to decide what to fix with the script, and what to fix manually in the source files. If you change source files, make sure that you've added a note about it in readme.md"

Reply:
done